### ISSUE TITLE
[ Feature Request ] Download progress indicator

### ISSUE THREAD

**Issue by SindreKjelsrud** [kepelet/flo#33]
Something like Spotify has where the ring around the download icon is like a progress indicator showing how far the download has come. Enhances user experience allowing users to easily track the progress of their downloads. :blush:

**Comment by faultables** [comment]
yeah this is on the list! and possibly can fix #17 too after resolving some download queue things

**Comment by faultables** [comment]
Resolved in 1.6
